API/Hardware_API.md

```diff
@@ -184,24 +184,93 @@ low\*     | pin voltage goes low  | Only available via pin.once()
 
 ### PWM pins
 
-**PWM pins** are pulse-width modulated pins ([wiki link](http://en.wikipedia.org/wiki/Pulse-width_modulation)). Essentially, PWM is a digital signal that spends between 0% and 100% of its time pulled high/on (this is its "duty cycle"). You can set the PWM pins to any value between 0 and 1 to approximate an analog signal. PWM is often used to control servo speeds or LED brightness.
+**PWM pins** are pulse-width modulated pins. Essentially, PWM is a digital signal that spends between 0% and 100% of its time pulled high/on (this is its "duty cycle"). You can set the PWM pins to any value between 0 and 1 to approximate an analog signal. PWM is often used to control servo speeds or LED brightness.
 
 Tessel has four PWM pins: pins 5 and 6 on each module port.
 
-Here is an example of setting a PWM pin:
+In the following example, the Tessel will change the color of an RGB LED by changing the duty cycle sent to each color pin over a set interval of time. 
```

Comment:
Might be confusing to discuss LED brightness via PWM and then discuss LED color via PWM